drivers/neural_net/intel_gna.c

```diff
@@ -460,12 +460,12 @@ static int intel_gna_infer(const struct device *dev,
 
 	/* copy input */
 	memcpy(handle->input, req->input, input_size);
-	SOC_DCACHE_FLUSH(handle->input, input_size);
+	z_xtensa_cache_flush(handle->input, input_size);
 
 	/* assign layer descriptor base address to configuration descriptor */
 	gna_config_desc.labase = (uint32_t)handle->vabase;
 	gna_config_desc.lacnt = (uint16_t)header->layer_count;
-	SOC_DCACHE_FLUSH(&gna_config_desc, sizeof(gna_config_desc));
+	z_xtensa_cache_flush(&gna_config_desc, sizeof(gna_config_desc));
```

Comment:
Why not remove the macros themselves as well?

Reply:
they are still being used outside of Zephyr..